docs/GettingStarted.asciidoc

```diff
@@ -264,6 +264,54 @@ Each click point can have an `id`, and if a needle contains multiple click point
 you must pass it to `testapi::assert_and_click` to select which click point
 to use.
 
+=== Configuration
+The different components of openQA read their configuration from the following
+files:
+
+* `openqa.ini`: This is the "web UI" config. Services providing the web
+  interface and related services such as the scheduler are configured via that
+  file.
+* `database.ini`: This file is also used by services providing the web interface
+  and related services such as the scheduler. It is used to configure how those
+  services connect to the database.
+* `workers.ini`: This file is used to configure the openQA worker including its
+  additional cache service.
+* `client.conf`: This file contains API credentials and is used by the openQA
+  worker and other tooling such as `openqa-cli` and `openqa-clone-job` to
+  authenticate with the web interface. One API key/secret can be configured per
+  web UI host.
+
+If these files are not present, defaults are used.
+
+Example configuration files are installed under `/usr/share/doc/openqa/examples`
+and in the
+https://github.com/os-autoinst/openQA/tree/master/etc/openqa[Git repository under `etc/openqa`].
+Continue reading the next sections for where you can place the actual
+configuration files and the possibility of creating drop-in configuration files.
+
+==== Locations
+All configuration files can be placed under `/etc/openqa`,
+e.g. `/etc/openqa/openqa.ini`. That is where administrators are expected to
+store system-wide configuration.
+
+Configuration files are also looked up under `/usr/etc/openqa` where a
+https://en.opensuse.org/openSUSE:Packaging_UsrEtc#Variant_1_(ideal_case)[package maintainer]
+can place default values deviating from upstream defaults.
+
+The client configuration can also be put under `~/.config/openqa/client.conf`.
+
+For development, checkout the section about
+<<Contributing.asciidoc#_customize_configuration_directory,customizing the configuration directory>>.
+
+==== Drop-in configurations
+It is possible to split the configuration into multiple files and store these
```

Comment:
Can we emphasize that drop-in configurations are preferred to the complete overrides? I am worried because now you *first* describe the complete ini files and then mention drop-in that people assume they would need or should use openqa.ini. WDYT?

Reply:
I would lean towards just mentioning the legacy config files which we continue to support at the end. That's what they are to my mind.